setpass/templates/helpdesk_ticket.txt

```diff
@@ -0,0 +1,7 @@
+[{username}]
```

Comment:
This seems off to me, to put something in the templates folder, when it isn't being used by the Flask template engine at all. And even on first glance it kinda looks like a Jinja template so that might be confusing.